tests/www/apply/test_list_prescriptions.py

```diff
@@ -652,3 +659,125 @@ def test_htmx_order(client):
+        job_seeker=job_seeker,
+        author_kind=AuthorKind.EMPLOYER,
+        author_siae=company,
+        author=company.members.first(),
+        criteria_kinds=[AdministrativeCriteriaKind.ASS, AdministrativeCriteriaKind.RSA],
+    )
+    no_criteria_prescriber_diag = IAEEligibilityDiagnosisFactory(
+        job_seeker=job_seeker,
+        author_kind=AuthorKind.PRESCRIBER,
+        author_prescriber_organization=prescriber_org,
+        author=prescriber,
+    )
+    prescriber_diag = IAEEligibilityDiagnosisFactory(
+        job_seeker=job_seeker,
+        author_kind=AuthorKind.PRESCRIBER,
+        author_prescriber_organization=prescriber_org,
+        author=prescriber,
+        criteria_kinds=[AdministrativeCriteriaKind.AAH, AdministrativeCriteriaKind.QPV],
+    )
+
+    prescriber_approval = ApprovalFactory(
+        user__first_name="Martine",
+        user__last_name="Martin",
+    )
+    employer_approval = ApprovalFactory(
+        user__first_name="Aline",
+        user__last_name="Bato",
+        with_diagnosis_from_employer=True,
+    )
+
+    job_applications = [
+        JobApplicationFactory(
+            state=JobApplicationState.NEW,
+            eligibility_diagnosis=None,
+            job_seeker__first_name="Pas de",
+            job_seeker__last_name="Diagnostique",
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.PROCESSING,
+            eligibility_diagnosis=company_diag,
+            job_seeker=job_seeker,
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.REFUSED,
+            eligibility_diagnosis=no_criteria_prescriber_diag,
+            job_seeker=job_seeker,
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.POSTPONED,
+            eligibility_diagnosis=prescriber_diag,
+            job_seeker=job_seeker,
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.ACCEPTED,
+            eligibility_diagnosis=employer_approval.eligibility_diagnosis,
+            job_seeker=employer_approval.user,
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.ACCEPTED,
+            eligibility_diagnosis=prescriber_approval.eligibility_diagnosis,
+            job_seeker=prescriber_approval.user,
+            to_company=company,
+            **common_kwargs,
+        ),
+        JobApplicationFactory(
+            state=JobApplicationState.ACCEPTED,
+            eligibility_diagnosis=prescriber_approval.eligibility_diagnosis,
+            job_seeker=prescriber_approval.user,
+            to_company__kind=CompanyKind.GEIQ,
+            **common_kwargs,
```

Comment:
```suggestion
            to_company__name="Tartempion",
            **common_kwargs,
```

Il faut figer to_company__name ici (une MAJ des snapshots va fixer les autres failing tests)